src/server/models/listItem/listItem.ts

```diff
@@ -478,14 +458,6 @@ export async function deleteListItem(id: number, userId: User["id"]): Promise<vo
           id,
         },
       }),
-      recordListItemEvent(
```

Comment:
I think this should stay - we want to record of a user deleted a list item

Reply:
Does that mean the EVENTS.DELETED line should be removed since there's no listItem for it to be connected to?

Reply:
I believe `recordListItemEvent` - confusingly - actually added an audit. That should stay, but for EVENTS.deleted, that can be removed